chainstate/test-framework/src/transaction_builder.rs

```diff
@@ -24,11 +24,10 @@ use rstest::rstest;
 
 /// The transaction builder.
 pub struct TransactionBuilder {
-    flags: u32,
+    flags: u128,
```

Comment:
Isn't it better to use compact encoding here? We could save 15 bytes when the flag is 0.

Reply:
That's exactly what I'm doing if you look at the previous commit. However, there's an issue in Python I'm still working on. Even going from master, changing this to compact and changing Python's TransactionV1 flags scalecodec schema to Compact<u32> makes the mempool tests fail because the transaction IDs are different... still trying to understand wth is going on.